src/user-guide/troubleshooting.rst

```diff
@@ -287,21 +289,111 @@ be possible to restart it as you would normally using ``cylc play``. Cylc
 will pick up where it left off.
 
 
-Why isn't my task running?
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Why is my task stuck in the waiting state - |task-waiting|?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tasks in the "waiting" state are waiting for something to happen before Cylc
+will submit them.
+
+To find out what is holding the task back from running:
+
+.. tab-set::
+
+   .. tab-item:: In the GUI
+      :sync: gui
+
+      Click on the task, then select "Info" from the menu.
+
+      This will open the "Info View" from which you can inspect the task's
+      state and prerequisites.
+
+      .. note::
```

Comment:
Link to https://github.com/cylc/cylc-ui/pull/2102 - note to self to remove this comment on merge of that PR.